### PR TITLE
Change documented options for os_networks_facts

### PR DESCRIPTION
**Pull request by G-Goldstein**
##### ISSUE TYPE

- Docs Pull Request
##### COMPONENT NAME

os_networks_facts
##### ANSIBLE VERSION

```
ansible 2.0.2.0
```
##### SUMMARY

os_network_facts currently lists 'network' as an available option, taking the Name or ID. In Ansible 2.0.2 to 2.2.0, this is not valid. Option 'name' or 'id' should be used instead.

```

```

### CODE REVIEW

**Comment by G-Goldstein** [comment]
The build failure here is environmental and not related to the innocuous documentation change I've made. I'm going to assume that it's not an issue unless I hear otherwise.

**Comment by G-Goldstein** [comment]
Depending on how the change should get into the Development branch, this pull request may be superseded by #3971

**Comment by gregdek** [comment]
Thanks @G-Goldstein. To the current maintainers, @emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/openstack/os_networks_facts.py [hunk 1]
```diff
@@ -33,10 +33,16 @@
...
     - "shade"
 options:
-   network:
+   name:
      description:
-        - Name or ID of the Network
+        - Name of the Network
      required: false
+     version_added: "2.2"
+   id:
```
Comment (rcarrillocruz) [inline cloud/openstack/os_networks_facts.py]: This parameter is not not used by the module, and is not needed.

The name parameter can accept either name or id.

File: cloud/openstack/os_networks_facts.py [hunk 2]
```diff
@@ -33,10 +33,16 @@
     - "python >= 2.6"
     - "shade"
 options:
-   network:
+   name:
      description:
-        - Name or ID of the Network
```
Comment (rcarrillocruz) [inline cloud/openstack/os_networks_facts.py:38]: the underlying method that the module calls to get networks facts, is shade search_networks:

https://github.com/openstack-infra/shade/blob/master/shade/openstackcloud.py#L1136

That method accepts either a name or an ID, and return it.

So you should leave 'Name or ID of the Network'

**Comment by rcarrillocruz** [comment]
needs_revision

**Comment by gregdek** [comment]
Thanks @G-Goldstein for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by G-Goldstein** [comment]
ready_for_review

**Comment by rcarrillocruz** [comment]
shipit

**Comment by gregdek** [comment]
Thanks again to @G-Goldstein for this PR, and thanks @emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request merged**